site/docs/downloads.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Downloads
+---
+
```

Comment:
you might also want to add a link to the KEYS and asc files so the releases can be verified.